testsuite/integration/basic/pom.xml

```diff
@@ -305,6 +305,7 @@
         <dependency>
             <groupId>xerces</groupId>
             <artifactId>xercesImpl</artifactId>
+            <version>2.12.0.SP05</version>
```

Comment:
why not just remove this dependency? any other test using this?

Reply:
Yes, there are 2 tests using package import from Xerces in testsuite/integration/basic TS @jbliznak

Reply:
Fixed @jbliznak - now 'provided' scope is used.